### PR TITLE
{Doc} Increase visibility of "Quoting issues with PowerShell"

### PR DESCRIPTION
**Pull request by jiasli**
**Description**  
The visibility of "_Quoting issues with PowerShell_" is still not high enough, leading to https://github.com/Azure/azure-cli/issues/13882.

Moving it to the top of the section.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S171

**Comment by yonzhan** [comment]
Doc

File: doc/use_cli_effectively.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ This becomes an issue because when the command shell (Bash, Zsh, Windows Command
 
 - Bash: [Quoting](https://www.gnu.org/software/bash/manual/html_node/Quoting.html)
 - PowerShell: [About Quoting Rules](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_quoting_rules)
+    - Due to a known issue [#1995](https://github.com/PowerShell/PowerShell/issues/1995) of PowerShell, some extra escaping rules apply, see [Quoting issues with PowerShell](quoting-issues-with-powershell.md) for more information.
```
Comment (qwordy) [inline doc/use_cli_effectively.md:131]:
```suggestion
    - Due to a known issue [#1995](https://github.com/PowerShell/PowerShell/issues/1995) of PowerShell, some extra escaping rules apply. See [Quoting issues with PowerShell](quoting-issues-with-powershell.md) for more information.
```

Comment (jiasli) [reply doc/use_cli_effectively.md]: Nice catch!

**Pull request merged**